doc/development_process/release_process.rst

```diff
@@ -98,6 +99,25 @@ Here is the description of the various moderation levels:
   - Bug Fixes: P1 and P2
   - Documentation + Test Coverage
 
+Release Quality Metrics
+***********************
+
+The current backlog of prioritized bugs shall be used as a quality metric to
```

Comment:
Better say "triaged bugs"?

Reply:
I think prioritized is better. We do a triage action to get to a prioritization disposition of the bugs and it is the prioritization states that matter in this guide. Just want to reinforce that to the reader.